Security-Operations/L4-Security-Analyst-Technical-Lead.md

```diff
@@ -0,0 +1,87 @@
+# L4: Security Analyst Technical Lead
```

Comment:
Is there a strong reason for the non-standard title? We tend to use "Lead XXX" as the role title for this level as a sensible default: `Lead Security Analyst`.

Reply:
I recall concern from the team a long time ago that the title `Lead Security Analyst` or `Lead Security Engineer` implied people leadership and that including the word `Technical` in the title would make it clear that it's a technical role and not a people management role. I also recently ran a poll in our team channel to see what name resonates the most with the team (https://octopusdeploy.slack.com/archives/C02A77EAYGZ/p1724655042872929).